lib/internal/test_runner/reporter/dot.js

```diff
@@ -1,24 +1,31 @@
 'use strict';
+
+const { green, red } = require('internal/util/colors');
 const { MathMax } = primordials;
 
 module.exports = async function* dot(source) {
   let count = 0;
   let columns = getLineLength();
+
   for await (const { type } of source) {
     if (type === 'test:pass') {
-      yield '.';
+      // Made green dot for pass
+      yield `${green}.`;
```

Comment:
@bantoinese83 based on [this example](https://github.com/nodejs/node/blob/db7d22dd700d826e7a4d153c276f6e127db847e5/lib/internal/main/watch_mode.js#L94) it would seem you need to set the color back to white after using it:
```suggestion
      yield `${green}.${white}`;
```

(Don't forget to import it and also fix `red`)

Reply:
@MoLow I assume you mean `clear`?
https://github.com/nodejs/node/blob/db7d22dd700d826e7a4d153c276f6e127db847e5/lib/internal/util/colors.js#L34